instat/dlgColumnStats.vb

```diff
@@ -150,14 +148,22 @@ Public Class dlgColumnStats
         TestOKEnabled()
     End Sub
 
-    Private Sub CoreControls_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverSelectedVariables.ControlContentsChanged, ucrReceiverByFactor.ControlContentsChanged, ucrChkPrintOutput.ControlContentsChanged, ucrChkStoreResults.ControlContentsChanged
-        TestOKEnabled()
-    End Sub
-
     Private Sub cmdProportionsPercentages_Click(sender As Object, e As EventArgs) Handles cmdProportionsPercentages.Click
         sdgProportionsPercentages.SetRFunction(clsDefaultFunction, bResetSubdialog)
         sdgProportionsPercentages.ShowDialog()
         bResetSubdialog = False
         TestOKEnabled()
     End Sub
+
+    Private Sub ucrChkPrintOutput_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrChkPrintOutput.ControlValueChanged
+        If ucrChkPrintOutput.Checked Then
+            ucrBase.clsRsyntax.iCallType = 2
+        Else
+            ucrBase.clsRsyntax.iCallType = 0
+        End If

```

Comment:
So here I've fixed the call type. Do you know the difference between call type 2 and call type 0?

If we always call an iCallType of 2 then we will get a double space in the output code if we aren't printing anything.

If we always call an iCallType of 0 then we will never get any output printed in the output window (asides from code!). Obviously if Print Output is checked then we want to see output so it will not always be an iCallType of 0.

If Print Output is checked, i.e. - if we want to see output in the output window, then we need an iCallType of 2 to view it.
Otherwise, we need an iCallType of 0 to avoid double spacing.

I know this wasn't anything you added, but just letting you know! :)